addons/website/static/src/snippets/s_facebook_page/000.js

```diff
@@ -3,6 +3,7 @@ odoo.define('website.s_facebook_page', function (require) {
 
 var publicWidget = require('web.public.widget');
 var utils = require('web.utils');
+const { debounce } = require("@web/core/utils/timing");
```

Comment:
take the opportunity to also replace the others `var` by `const` ;)

Reply:
@bvr-odoo I usually don't do that otherwise we will have unnecessary diff. noise but as you are suggesting so we will do it.

Reply:
Hello @bvr-odoo 
As we are rendering the iframe on resizing, it will trigger an event on every instance of time to render. So, Debounce will delay the method to trigger for about a minimal amount of time on every last event call. In the render function, we are fetching a URL that takes time to render and resize the iframe.

While in throttling, the rendering method would still be triggered at regular intervals during the resize process. This means that even if the resize events occur very frequently, the iframe would still be rendered periodically, leading to multiple renders while resizing is still in progress.

Reply:
> @bvr-odoo I usually don't do that otherwise we will have unnecessary diff. noise but as you are suggesting so we will do it.

It was not a good idea sorry, You can keep it like that